scripts/build/Jenkins/Jenkinsfile

```diff
@@ -750,6 +750,14 @@ finally {
             } else {
                 buildFailure = tm('${BUILD_FAILURE_ANALYZER}')
                 emailBody = "${BUILD_URL}\n${buildFailure}!"
+                if(env.SNS_TOPIC_GITHUB_ISSUE) {
+                    message_json = ["repository_name":env.REPOSITORY_NAME, "branch_name":env.BRANCH_NAME, "build_failure":buildFailure]
+                    snsPublish(
+                        topicArn: env.SNS_TOPIC_GITHUB_ISSUE,
+                        subject:'Build Result',
```

Comment:
Should/could the subject be 'Build Failure' since that is a more specific type of result? Or, is the subject supposed to match the one used above for env.SNS_TOPIC

Reply:
Sure, I will change the subject to 'Build Failure'.